x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackClientPlugin.java

```diff
@@ -247,9 +246,6 @@ public List<Setting<?>> getSettings() {
         // TODO split these settings up
         settings.addAll(XPackSettings.getAllSettings());
 
-        settings.add(LicenseSettings.SELF_GENERATED_LICENSE_TYPE);
-        settings.add(LicenseSettings.ALLOWED_LICENSE_TYPES_SETTING);
```

Comment:
GitHub does not allow me to comment on the above line 244
```java
settings.add(Setting.groupSetting("license.", Setting.Property.NodeScope));
```
Do we even know what this group setting is for? I suspect we don't need it for serverless.

Reply:
> Do we even know what this group setting is for? I suspect we don't need it for serverless.

Ha, I looked into that and could not figure out that group setting was for. It was introduced a long time ago and I guess it reserves that namespace and maybe it was used a long time ago ? We should be probably remove it but it's existence should be harmless.